src/components/plugins/DummyPlugin/DummyPlugin.test.jsx

```diff
@@ -7,11 +8,13 @@ import DummyPlugin from './DummyPlugin';
 describe('Testing dummy plugin', () => {
   test('should render props and text elements test for the page component', () => {
     const { getByText } = render(
-      <BrowserRouter>
-        <Provider store={store}>
-          <DummyPlugin />
-        </Provider>
-      </BrowserRouter>
+      <MockedProvider>
```

Comment:
there is no test written here

Reply:
In DummyPlugin.tsx file only a single statement is present as you can see in the image: 

![image](https://user-images.githubusercontent.com/63240102/183301571-cf3d4fa0-7baf-48c6-ae40-18aea5b6c25e.png)

So, only one assertion is done.